include/aspect/particle/particle.h

```diff
@@ -121,6 +121,21 @@ namespace aspect
                   const types::particle_index &new_id);
 
         /**
+         * Constructor for Particle, creates a particle with the specified
+         * ID at the specified location. Note that Aspect
+         * does not check for duplicate particle IDs so the generator must
+         * make sure the IDs are unique over all processes.
+         *
+         * @param[in] new_location Initial location of particle.
+         * @param[in] new_reference_location Initial location of the particle
+         * in the coordinate system of the reference cell.
+         * @param[in] new_id Globally unique ID number of particle.
+         */
+        Particle (const Point<dim> &new_location,
+                  const Point<dim> &new_reference_location,
+                  const types::particle_index &new_id);
```

Comment:
Isn't `particle_index` just some plain old integer? If it's a scalar, just passing it by const value is fine, no need for a reference.